p11-kit/rpc-transport.c

```diff
@@ -439,7 +440,7 @@ rpc_socket_read (rpc_socket *sock,
 			/* We ignore the options, so read into the same as buffer */
 			if (!p11_buffer_reset (buffer, sock->read_olen) ||
 			    !p11_buffer_reset (buffer, sock->read_dlen)) {
-				warn_if_reached ();
+				warn_if_reached (;);
```

Comment:
I think a better fix would be remove argument (`v`) from the `warn_if_reached` definition in common/debug.h.